bench/perf_run_py/perf_run.py

```diff
@@ -75,18 +79,16 @@ def get_git_info(git_working_tree_dir):
     return git_info
 
 
-def construct_wtperf_command_line(wtperf: str, env: str, test: str, home: str, arguments: List[str]):
+def construct_command_line(exec_path: str, test: str, home: str, arguments: List[str]):
     command_line = []
-    if env is not None:
-        command_line.append(env)
-    command_line.append(wtperf)
+    command_line.append(exec_path)
     if test is not None:
-        command_line.append('-O')
+        command_line.append('-O') if is_wtperf_test(exec_path) else None
         command_line.append(test)
     if arguments is not None:
         command_line.extend(arguments)
     if home is not None:
-        command_line.append('-h')
+        command_line.append('-h') if is_wtperf_test(exec_path) else command_line.append('--home')
         command_line.append(home)
     return command_line
 
```

Comment:
The PEP8 coding standard is for two blank lines before a global function.

Reply:
Made the python scripts PEP8 compliant - [14f2a2c](https://github.com/wiredtiger/wiredtiger/pull/7223/commits/14f2a2c18e17c5135229c5542a7bb7c9a9c93b3e)